dataservice/api/common/model.py

```diff
@@ -10,6 +10,9 @@
 from dataservice.extensions.flask_indexd import RecordNotFound
 from dataservice.api.common.id_service import uuid_generator, kf_id_generator
 
+COMMON_ENUM = {"Not Reported", "Not Applicable", "Not Allowed To Collect",
```

Comment:
Needs "No Match" for (eg ontologies) when a specific domain exists but the correlated data does not match anything within the domain

Reply:
👍 i think it is better to add `No Match` for the entities diagnosis, phenotype and biospecimens in their respective schemas

Reply:
I agree w @parimalak. `No Match` doesn't pertain to all enums so should not be included in common enums